index.html

```diff
@@ -21,7 +21,7 @@
 
             <input id='rgbInput' type="text" placeholder=" rgb(201, 148, 187)">
             <input id="hexInput" type="text" placeholder=" #c994bb">
-            <button id="random-color-btn">Generate random color</button>
+            <button id="random_color_btn">Generate random color</button>
```

Comment:
I suggest consistency with ids and class names, i.e. all `camelCase` or `with-dashes` or `snake_case`